RealmModelGenerator/AttributesView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  AttributesView.swift
+//  RealmModelGenerator
+//
+//  Created by Zhaolong Zhong on 3/31/16.
+//  Copyright © 2016 QuarkWorks. All rights reserved.
+//
+
+import Cocoa
+
+@objc protocol AttributesViewDataSource {
+    optional func numberOfRowsInAttributesView(attributesView:AttributesView) -> Int
+    optional func attributesView(attributesView:AttributesView, titleForAttributeAtIndex index:Int) -> String?
+}
+
+@objc protocol AttributesViewDelegate {
+    optional func addAttributeInAttributesView(attributesView:AttributesView)
+    optional func attributesView(attributesView:AttributesView, removeAttributeAtIndex index:Int)
+    optional func attributesView(attributesView:AttributesView, selectionChange index:Int)
+    optional func attributesView(attributesView:AttributesView, shouldChangeAttributeName name:String,
+        atIndex index:Int) -> Bool
+}
+
+@IBDesignable
+class AttributesView: NibDesignableView, NSTableViewDelegate, NSTableViewDataSource, TitleCellDelegate {
+    static let TAG = NSStringFromClass(AttributesView)
+    
+    @IBOutlet var tableView:NSTableView!
+    @IBOutlet weak var addButton: NSButton!
+    @IBOutlet weak var removeButton: NSButton!
+    
+    weak var dataSource:AttributesViewDataSource?
```

Comment:
setting the dataSource should call a function in the view that is called reloadData()